Aztec/Classes/TextKit/LayoutManager.swift

```diff
@@ -233,8 +237,24 @@ private extension LayoutManager {
                     start = textStorage.numberOfItems(in: list, at: enclosingRange.location)
                 }
             }
+
+            var indentLevel = 1
+            // Determine indentation level, if needed. The indentation level is only used by the standard list style
+            if list.style == .unordered, listIndentStyle == .standard {
+                // only get the width of the first level once
+                if firstLevelWidth == nil {
+                    firstLevelWidth = paragraphStyle.indentToFirst(TextList.self)
+                }
+
+                // calculate current indent level
+                let indentWidth = paragraphStyle.indentToLast(TextList.self)
+                if let firstLevelWidth = firstLevelWidth {
+                    indentLevel = Int(indentWidth / firstLevelWidth)
+                }
+            }
+
             markerNumber += start
-            let markerString = list.style.markerText(forItemNumber: markerNumber)
+            let markerString = list.style.markerText(forItemNumber: markerNumber, indentLevel: indentLevel)
```

Comment:
Nit: The `indentLevel` isn't accurate if the list style isn't unordered + standard, which could cause confusion.